pkg/render/tiers/tiers.go

```diff
@@ -52,7 +59,9 @@ func Tiers(cfg *Config) render.Component {
 }
 
 type Config struct {
-	Openshift bool
+	Openshift    bool
+	NodeLocalDNS bool
```

Comment:
To simplify this a little bit, I don't think we need both `NodeLocalDNS` as well as `KubeDNSCIDR`, because right now they are bound to each other. You'll never have NodeLocalDNS set to `false` and `KubeDNSCIDR` set to an IP, and vice-versa. 

We should be able to do something like this:

```go
type Config struct {
  Openshift bool
  DNSEgressIP string
}
```

and then 

```go

if DNSEgressIP != "" {
  // Add the policy for DNS egress
} else {
  // Remove the policy for DNS egress
}
```

This just simplifies the logic a tiny bit and removes the ability to specify "invalid" combinations.